frontend/src/state/actions.ts

```diff
@@ -0,0 +1,12 @@
+import { action } from 'typesafe-actions';
+import {Person, Request, User} from './index';
+
+export const enum ActionTypes {
+    UPDATE_USER = 'UPDATE_USER',
+    UPDATE_PEOPLE = 'UPDATE_PEOPLE',
+    UPDATE_REQUESTS = 'UPDATE_REQUESTS'
```

Comment:
CONSISNTNEJNEJKFHDGFKHG QUOTESSSSSSSSSSSSSS

Reply:
Has been fixed my fine fellow

Reply:
#31 will help with this!